Doc/library/fractions.rst

```diff
@@ -14,8 +14,8 @@
 The :mod:`fractions` module provides support for rational number arithmetic.
 
 
-A Fraction instance can be constructed from a pair of integers, from
-another rational number, or from a string.
+A Fraction instance can be constructed from a pair of numbers, from
+another number, or from a string.
```

Comment:
What do you think about this?
```suggestion
A Fraction instance can be constructed from a pair of rational numbers, from
a single number, or from a string.
```

Reply:
Indeed, this better describes two-arg case.